### DISCUSSION TITLE
PC to steam deck

### DISCUSSION THREAD

**Discussion by BroodingMawlek** [mtkennerly/ludusavi#275] [Q&A] [answered]
Hi, I have configured ludisavi PC to backup to a local file and dropbox/

On Steam deck I go into restore mode and set source to 
/home/deck/Dropbox/ludusavi-backup

The file is then copied back to the windows path 
/home/deck/C_/Users/User/AppData/Roaming/DarkSoulsIII/

I need it in the Linux dir
/home/deck/.local/share/Steam/steamapps/compatdata/374320/pfx/drive_c/users/steamuser/AppData/Roaming/DarkSoulsIII/

Am I doing something wrong, any tips?

Thanks

**Comment by mtkennerly** [comment, marked as answer]
Hi! Unfortunately, right now, this would require setting up a lot of redirects for each game. I have some ideas for Ludusavi to make this easier and support it automatically, but it's not implemented yet. Please keep an eye on https://github.com/mtkennerly/ludusavi/issues/194 for any updates.

**Comment by imfm** [comment]
I have the same issue. This could be a killer feature as there are a surprising amount of Steam games that do not support cloud saves and the only way to synchronize between PC and steam deck is to manually copy the files. It would be great if Ludusavi could be configured to copy files to a base folder (i.e. "**<base_folder>**/Users/User/AppData/Roaming/") as opposed assuming a folder path and taking the whole thing with each file (i.e. "**/home/deck/C_**/Users/User/AppData/Roaming/"). Also, would be nice to have separate configurations for Windows/Linux.
